addons/stock_account/models/product.py

```diff
@@ -114,11 +109,6 @@ class ProductProduct(models.Model):
     stock_value = fields.Float(
         'Value', compute='_compute_stock_value')
 
-    @api.onchange('type')
-    def onchange_type_valuation(self):
-        # TO REMOVE IN MASTER
-        pass
-
     @api.multi
     def do_change_standard_price(self, new_price, account_id):
         """ Changes the Standard Price of Product and creates an account move accordingly."""
```

Comment:
@jco-odoo If you ask me an opinion, I personally prefer stay with this unmerged until saas-18 with full review test than left known technical debts that will clearly give problems in saas-17.

 prefer have this very first.

But if this become a perfonrmance problem this can be fixed in the same stable (-17) after merge, because is not a matter of logic.

The performance problem WILL be there.

If you have for example a little business with 20 locations (that's little size if you want to have precise control of locations) you can see the problem easy.

And about the product in the account_move_line AFAICR historically this was always there even on the name, but we must include the traceability data (even if reports to use the daa come afterwards).